compiler/rustc_error_messages/locales/en-US/expand.ftl

```diff
@@ -3,3 +3,12 @@ expand_explain_doc_comment_outer =
 
 expand_explain_doc_comment_inner =
     inner doc comments expand to `#![doc = "..."]`, which is what this macro attempted to match
+
+expand_expr_repeat_no_syntax_vars = 
+    attempted to repeat an expression containing no syntax variables matched as repeating at this depth
+
+expand_must_repeat_once = 
+    this must repeat at least once
+
+count_repetition_misplaced =
```

Comment:
Oh, and the slugs here should start with `expand_` in this file.

Reply:
Ops, my bad. Solved in the latest commit.